src/area-matrix.c

```diff
@@ -2,6 +2,7 @@
 #include "napi_utils.h"
 #include "control.h"
 #include "events.h"
+#include "values.h"
 
 static const char *MODULE = "AreaMatrix";
 
```

Comment:
free_matrix? or we could also use the same function for all these simple cases...

Reply:
There are only two simple cases: matrix and path. Brush and stroke need to free an array as well.

Reply:
Yes, I see... let's go with free_matrix